deploy/k8m.yaml

```
@@ -162,10 +173,16 @@ spec:
             #   value: "true"
             # #管理员用户名
             # - name: ADMIN_USERNAME
-            #   value: "admin"
+            #   valueFrom: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)